src/components/Input.react.js

```diff
@@ -21,15 +21,22 @@ export default class Input extends Component {
     render() {
         const {
             fireEvent,
-            setProps
+            setProps,
+            inputType
```

Comment:
I believe that this should be `type` to match https://github.com/plotly/dash-core-components/pull/100/files#diff-368d71637f90189b9dab46fb80b2c1f1R78